pkgs/development/tools/tabnine/default.nix

```diff
@@ -1,19 +1,20 @@
 { stdenv, lib, fetchurl, unzip }:
 
 let
-  version = "3.5.15";
+  # You can check the latest version with `curl -sS https://update.tabnine.com/bundles/version`
```

Comment:
Could we write an update script for this?

Reply:
I wasn't sure how to write one that would correctly update the fetcher for darwin and linux